### ISSUE TITLE
Autocompletion for commands

### ISSUE THREAD

**Issue by hediet** [fabiospampinato/vscode-commands#10]
I don't know how easy it is, but I would make configuring things easier a lot.

This is what I mean:
![image](https://user-images.githubusercontent.com/2931520/52567902-ac333380-2e0d-11e9-8475-96858b5f9e6f.png)

This could be done by either implementing a small, additional language server for json files or maybe even by just providing a json schema. Maybe one could even open an issue on the vscode repository itself to support autocompleting commands by setting a custom schema property.

**Comment by fabiospampinato** [comment]
I don't think this is implementable, as we can't generate the schema dynamically but the list of commands changes dynamically (as there are commands provided by extensions).
